crates/vm/levm/src/errors.rs

```diff
@@ -67,7 +67,15 @@ pub enum VMError {
     #[error("Remaining Gas Underflow")]
     RemainingGasUnderflow, // When gas used is higher than gas limit, is there already an error for that?
 
-    // OutOfGas
+    #[error("Out Of Gas")]
+    OutOfGasErr(#[from] OutOfGasError),
```

Comment:
It is ok to name it `OutOfGas` here.
```suggestion
    OutOfGas(#[from] OutOfGasError),
```

Reply:
Yes, done in [Remove empty out of gas and reuse the name](https://github.com/lambdaclass/lambda_ethereum_rust/pull/1138/commits/62dca9cc4231f1c440c59f2ec4391a7158def149)